README.md

```diff
@@ -393,10 +393,10 @@ openapi-generator version
 ```
 
 <!-- RELEASE_VERSION -->
-Or install a particular OpenAPI Generator version (e.g. v4.2.2):
+Or install a particular OpenAPI Generator version (e.g. v4.2.3):
```

Comment:
@jimschubert FYI. I intentionally left this one since I want to show how to install an old version instead of the latest version but I'm ok showing whatever version (latest or old) in the doc.

Reply:
K. If that's the intent, please change and remove the `RELEASE_VERSION` markers around the section. Although, I think users could infer this applies for all versions.